### PR TITLE
Introduce optuna-dashboard on README.

### PR DESCRIPTION
**Pull request by c-bata**
## Motivation

| Manage studies | Visualize with interactive graphs |
| -------------- | --------------------------------- |
| manage-studies | optuna-realtime-graph |

I think `optuna-dashboard` is useful for most of Optuna users, but it is still not well known.
https://github.com/optuna/optuna-dashboard

## Description of the changes

Introduce optuna-dashboard on README

### CODE REVIEW

**Comment by toshihikoyanase** [comment]
@Crissman @HideakiImamura Could you review this PR, please?

File: README.md [hunk 1]
```diff
@@ -106,6 +106,27 @@ study.optimize(objective, n_trials=100)  # Invoke optimization of the objective
 * [tf.keras](./examples/pruning/tfkeras_integration.py)
 * [XGBoost](./examples/pruning/xgboost_integration.py)
 
+
+## Web Dashboard (experimental)
+
+The new Web dashboard is under the development at [optuna-dashboard](https://github.com/optuna/optuna-dashboard).
+It is still experimental, but much better in many regards.
+Any feature requests and bug reports are welcome!
```
Comment (Crissman) [inline README.md:114]:
```suggestion
Feature requests and bug reports welcome!
```

File: README.md [hunk 2]
```diff
@@ -106,6 +106,27 @@ study.optimize(objective, n_trials=100)  # Invoke optimization of the objective
...
+
+| Manage studies | Visualize with interactive graphs |
+| -------------- | --------------------------------- |
+| ![manage-studies](https://user-images.githubusercontent.com/5564044/97099702-4107be80-16cf-11eb-9d97-f5ceec98ce52.gif) | ![optuna-realtime-graph](https://user-images.githubusercontent.com/5564044/97099797-66e19300-16d0-11eb-826c-6977e3941fb0.gif) |
+
+You can install `optuna-dashboard` via pip and the usage is like this:
```
Comment (Crissman) [inline README.md:120]:
```suggestion
Install `optuna-dashboard` via pip:
```

**Review by Crissman — CHANGES_REQUESTED** [review]
Good idea!

**Comment by c-bata** [comment]
Thank you! I applied your suggestions.

**Review by HideakiImamura — APPROVED** [review]
Totally agree. LGTM!

**Review by Crissman — APPROVED** [review]

**Comment by c-bata** [comment]
Can we merge this since there are two approves?

**Comment by toshihikoyanase** [comment]
Yes, we can.
@Crissman Could you merge this PR, please?

**Pull request merged**